src/lib/checkbox/checkbox.ts

```diff
@@ -88,6 +89,9 @@ export class MdCheckbox implements ControlValueAccessor {
   /** A unique id for the checkbox. If one is not supplied, it is auto-generated. */
   @Input() id: string = `md-checkbox-${++nextId}`;
 
+  /** Whether the ripple effect on click should be disabled. */
+  @Input() @BooleanFieldValue() disableRipple: boolean = false;
```

Comment:
I just removed `@BooleanFieldValue` in another PR- custom decorators like this caused tree-shaking not to work. The replacement is a function called `coerceBooleanProperty`